subprojects/docs/src/samples/userguide/files/archivesChangedBaseName/kotlin/build.gradle.kts

```diff
@@ -0,0 +1,24 @@
+plugins {
+    base
+}
+
+version = "1.0"
+project.setProperty("archivesBaseName", "gradle")
```

Comment:
Good to know. The groovy code just uses `archivesBaseName = "gradle"`, so that wasn't obvious to me. I'm surprised that, apparently, doing the same thing in groovy doesn't work, though.

Reply:
@eskatos I don't see any reference to `base` in the migration guide for the Kotlin DSL. I think it would be sensible to add something if possible. There's no reference to it in the [Building Kotlin JVM Libraries](https://guides.gradle.org/building-kotlin-jvm-libraries/) guide either.